### PR TITLE
Voodoo cam tuneup

### PR DESCRIPTION
**Pull request by Kubius**
[FEAT][MINOR]
## About the PR 

Fixes up my old voodoo camera code to be less messed up and a *little* more balanced.

- Voodoo photo interactions now properly respect COMBAT_CLICK_DELAY.
- Default enchantment strength of photos reduced from 66 interactions to 13.
- Voodoo photos now crumble to dust when their enchantment runs out.
- Added an additional voodoo camera power level (and associated typed variant) that creates photos with an unbreakable enchantment.
- Photo-throwing now has flavor text at the target end.
- Fart telepresence now uses the correct text color.

## Why's this needed? 

In the rare event that the voodoo camera is actually used, this makes it work much better.

### CODE REVIEW

File: code/obj/item/cameras.dm [hunk 1]
```diff
@@ -177,22 +182,34 @@
...
-		if (enchant_power && cursed_dude && istype(cursed_dude, /mob))
+		if (enchant_power && world.time > src.enchant_delay && cursed_dude && istype(cursed_dude, /mob))
 			cursed_dude.attackby(W,user)
+			src.enchant_delay = world.time + COMBAT_CLICK_DELAY
 			if(enchant_power > 0) enchant_power--
 		else
 			..()
+		if(enchant_power == 0)
+			boutput(user,"<span style=\"color:red\"><b>[src]</b> crumbles away to dust!</span>")
+			SPAWN_DBG(0)
```
Comment (mybluecorners) [inline code/obj/item/cameras.dm:199]: Not necessary to spawn a qdel()

File: code/obj/item/cameras.dm [hunk 2]
```diff
@@ -177,22 +182,34 @@
...
+				qdel(src)
 		return
 
 	throw_begin(atom/target)
-		if (enchant_power && cursed_dude && istype(cursed_dude, /mob))
+		if (enchant_power && world.time > src.enchant_delay && cursed_dude && istype(cursed_dude, /mob))
```
Comment (mybluecorners) [inline code/obj/item/cameras.dm:204]:
```suggestion
		if (enchant_power && world.time > src.enchant_delay && cursed_dude && ismob(cursed_dude))
```

File: code/obj/item/cameras.dm [hunk 3]
```diff
@@ -177,22 +182,34 @@
...
+			cursed_dude.visible_message("<span style=\"color:red\"><b>[cursed_dude] is violently thrown by an unseen force!</b></span>")
 			cursed_dude.throw_at(get_edge_cheap(src, get_dir(src, target)), 20, 1)
+			src.enchant_delay = world.time + COMBAT_CLICK_DELAY
 			if(enchant_power > 0) enchant_power--
+		if(enchant_power == 0)
+			src.visible_message("<span style=\"color:red\"><b>[src]</b> crumbles away to dust!</span>")
+			SPAWN_DBG(0)
```
Comment (mybluecorners) [inline code/obj/item/cameras.dm:211]: another unnecessary spawn

**Comment by Kubius** [comment]
Things straightened out as requested, hopefully. Heck

**Pull request merged**